uint64 timestamp				# Microseconds since system boot
bool vtol_in_rw_mode			# true: vtol vehicle is in rotating wing mode
bool vtol_in_trans_mode
bool vtol_transition_failsafe	# vtol in transition failsafe mode
bool fw_permanent_stab			# In fw mode stabilize attitude even if in manual mode
float32 airspeed_tot			# Estimated airspeed over control surfaces
